cmd/object-api-utils.go

```diff
@@ -168,7 +168,7 @@ func pathJoin(elem ...string) string {
 func mustGetUUID() string {
```

Comment:
Maybe we need to add context to this function. If we decide to do, we can do it in a different PR

Reply:
OK.